addons/mail/static/src/variables.scss

```diff
@@ -12,3 +15,20 @@ $o-mail-discuss-sidebar-scrollbar-width: 15px !default;
 
 $o-mail-moderation-accept-color: theme-color('success') !default;
 $o-mail-moderation-reject-color: theme-color('danger') !default;
+
+$o-mail-thread-avatar-size: 36px !default;
+$o-mail-thread-icon-opacity: 0.6 !default;
+$o-mail-thread-side-date-opacity: 0.6 !default;
+$o-mail-thread-window-bg: #FAFAFA !default;
+$o-mail-thread-window-width: 325px !default;
+$o-mail-chatter-gap: 10px !default;
+$o-mail-chatter-mobile-gap: 2% !default;
+$o-mail-chat-header-height: 46px !default;
+$o-mail-attachment-image-size: 100px !default;
+$o-mail-sidebar-icon-opacity: 0.7 !default;
+$o-mail-chat-sidebar-width: 250px !default;
+$o-mail-partner-avatar-size: 24px !default;
```

Comment:
I think we should introduce conventions in this file, because it becomes a mess.
Something like alphabetical order + comment above each variable.

Reply:
alphabetical does not work because one variable could use another and it must be defined before

At best we should split by "scope" and then inside the scope we might want to apply alphabetical when only when it makes sense.